### DISCUSSION TITLE
基地台被拆，為何整個里的手機訊號全滅?

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#828] [General]
假新聞的負面影響包括讓你家訊號變差？基地台的電磁波是否對人健康造成影響根本沒有科學依據，這不過就是可悲媒體捏造出來博眼球的話題罷了。絕命律師主角的哥哥患有心理疾病，一直覺得自己對電磁波過敏，這不免讓我聯想，當地居民是不是集體罹患類似的心理疾病？
[文章連結](https://www.kocpc.com.tw/archives/467297)

**Comment by ru0372572** [comment]
想知道這之後當地居民、電信業者還有學校會怎麼做處理
順帶一提，絕命毒師也很好看~~~

**Comment by ruthchen1020** [comment]
所以有人證實電磁波會對人體有傷害嗎?
如果心臟有裝甚麼電子器械的話真的可能會被影響。

**Comment by StevenWanglolz** [comment]
電磁波對人體是否有傷害好像是一直存在的爭議，現在又有各種無線充電的裝置，能不能有人給個肯定的解答啊!
